semgrep/semgrep/semgrep_main.py

```diff
@@ -131,39 +131,27 @@ def rename_rule_ids(valid_configs: Dict[str, Any]) -> Dict[str, Any]:
 ### Handle output
 
 
-def post_output(output_url: str, output_data: Dict[str, Any]) -> None:
+def post_output(output_url: str, output: str) -> None:
     import requests  # here for faster startup times
 
     print_msg(f"posting to {output_url}...")
-    r = requests.post(output_url, json=output_data)
+    r = requests.post(output_url, data=output)
```

Comment:
wish we did the serialization closer to the post but I suspect you want to do this once